v2/community/supertokens-core/self-hosted/api-keys.mdx

```diff
@@ -17,6 +17,8 @@ import TabItem from '@theme/TabItem';
 This is only relevant if you are running SuperTokens on your own.
```

Comment:
Change this to "This is only relevant if you are running SuperTokens core on your own."